sdks/python/apache_beam/runners/dataflow/internal/apiclient.py

```diff
@@ -186,8 +186,13 @@ def __init__(self, packages, options, environment_version, pipeline_url):
     if job_type.startswith('FNAPI_'):
       self.debug_options = self.debug_options or DebugOptions()
       self.debug_options.experiments = self.debug_options.experiments or []
-      if not self.debug_options.lookup_experiment(
-          'runner_harness_container_image'):
+      if self.debug_options.lookup_experiment(
```

Comment:
Few thoughts here 
1. I think we cannot instantiate `DebugOptions()` in `self.debug_options = self.debug_options or DebugOptions()`. We should create a view of existing `options`.  All views of pipeline options should be views of the same underlying object, and I think that line will break it. Do you know why we added it?
2. For the purpose of this PR it should be sufficient to pass `options` into `_use_unified_worker()` in line 190.

Reply:
I agree. I think I added it because in certain case it was passed in test. I might be wrong though. 

Makes sense. I will update the PR with both these suggestions.